### ISSUE TITLE
CLI: Warning in output

### ISSUE THREAD

**Issue by kijerk** [languagetool-org/languagetool#1497]
Since LT version 4.5 I get the following warning as the first line  when using the command line:

```
15:59:09.076 [main] WARN org.languagetool.rules.spelling.suggestions.XGBoostSuggestionsOrderer - At the moment, your platform (Windows) is not supported by the official XGBoost maven package; ML-based suggestion reordering is disabled.
```
In both cases, using the `--api` or `--json` switch.

This renders at least the xml invalid. Can I turn off the ordering feature, if it there is no support for Windows? Or please remove the warning from the output...

**Comment by kijerk** [comment]
@fabrichter I have seen that the message is introduced by #1469.

Is there a simple way to deactivate the ordering on Windows? Or do I have to remove the message manually from the output? Are there any plans for this in the near future?

**Comment by danielnaber** [comment]
Shouldn't logger.warn end up on `stderr` instead of `stdout`? Then you could filter it (`2>/dev/null` on Linux, no idea how it works on Windows).

**Comment by kijerk** [comment]
Yes, you should be right... I will try to explicitly pipe `stout` and will give feedback on monday.

**Comment by kijerk** [comment]
Just had time for some testing...

`17:52:51.793 [main] WARN org.languagetool.rules.spelling.suggestions.XGBoostSuggestionsOrderer...` is the first output on `stout` (before the real LT output...).

To `stderr` only the following message ist written: `Working on {FILEPATH}...` (if there are no additional errors...).

So one solution would be to get the logger to write the xgboost error the same as the `Working on {FILEPATH}...` output.
